_src/m2-sku-reindexing-form.md

```diff
@@ -0,0 +1,44 @@
+---
+layout: m2-documentation
+title: SKU reindexing form
+permalink: /doc/m2/sku-reindexing-form/
+description: Learn about SKU reindexing form in the Algolia extension for Magento 2
+---
+
+## Access
+
+This feature can be accessed directly from the Magento2 back-office main menu :
+**Stores > Algolia Search > Reindex SKU(s)**
```

Comment:
It could be good for Algolian to have the keyword SKU in [our glossary](https://glossary.algolia.com/?q=sku), even if that glossary is internal only.